frontend/src/components/dashboard/aliases/BlockLevelSlider.module.scss

```diff
@@ -143,6 +176,8 @@ $trackLineHeight: 4px;
           // offset it by half of the width of the track, minus half its own
           // width:
           left: calc(50% - var(--thumbDiameter) / 2);
+          top: $iconHeight;
+          cursor: pointer;
```

Comment:
This is new, and needed because `.track-stop-promotional` no longer automatically got positioned at the right height by virtue of being part of the `.track`.

Additionally, since  the entire track is clickable (to move the slider), it already had `cursor: pointer`, so now that `.track-stop-promotional` is outside of the track, it needed its own.